.ci_support/linux_64_.yaml

```diff
@@ -14,7 +14,11 @@ cxx_compiler_version:
 - '10'
 docker_image:
 - quay.io/condaforge/linux-anvil-cos7-x86_64
+lz4_c:
+- 1.9.3
 pin_run_as_build:
+  lz4-c:
+    max_pin: x.x.x
```

Comment:
Interesting. Would have thought PR ( https://github.com/conda-forge/blosc-feedstock/pull/53 ) would have gotten this as it did the other dependencies when re-rendering.

Reply:
We should delete this right: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml#L266

Reply:
https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/2860